### DISCUSSION TITLE
UI improvement - Rerollbot

### DISCUSSION THREAD

**Discussion by minerop5000** [NiHoel/Anno1800UXEnhancer#23] [Ideas]
Hi,
thanks for the amazing Rerollbot.
As a smal improvment i would like to request to grey out the number input, if the item cannot be rolled at any npc.

**Comment by NiHoel** [comment]
@Veraatversus added the input fields to her AssetViewer. 
My application then gets the numbers and checks which items can be bought. So it requires some work on the side of the AssetViewer to implement this.

**Comment by NiHoel** [comment]
Implemented with v7.1
By default, only purchasable items are now shown.
